conbench/tests/containers/dex/config.yml

```diff
@@ -0,0 +1,51 @@
+# issuer: http://127.0.0.1:5556/dex-for-conbench
+# The DNS name `dex` is provided by e.g. docker-compose
+issuer: http://dex:5556/dex-for-conbench
+# will expose http://127.0.0.1:5556/dex/.well-known/openid-configuration
+
+storage:
+  type: sqlite3
+  # config:
+  #   file: /dex-conbench-tests.db
```

Comment:
Personally I like when some of those stay around so that one can quickly change things w/o having to consult docs too much, but I am also happy with doing a proper cleanup :).

Reply:
Yes, agreed! We can keep these around for that purpose. I'm just not quite sure after reading these comments what sorts of features they might enable or why we might want to do that in the future.